debezium-connector-jdbc/src/test/java/io/debezium/connector/jdbc/e2e/JdbcSinkPipelineToPostgresIT.java

```diff
@@ -216,4 +217,47 @@ public void testTimestampWithTimeZoneDataTypeWithInfinityValue(Source source, Si
                 },
                 ResultSet::getString);
     }
+
+    @TestTemplate
+    @ForSource(value = SourceType.POSTGRES, reason = "The SPARSEVEC data type only applies to PostgreSQL")
+    @WithPostgresExtension("vector")
+    public void testSparseVectorDataType(Source source, Sink sink) throws Exception {
+        assertDataTypeNonKeyOnly(source,
+                sink,
+                "sparsevec(25)",
+                List.of("'{1:0.1,3:0.2,5:0.3}/25'"),
+                List.of("{1:0.1,3:0.2,5:0.3}/25"),
+                (config) -> config.with("include.unknown.datatypes", true),
```

Comment:
Is this really needed? If yes then there must be an issue wih source connector. Same for other methods,

Reply:
Likely not, product of cut-n-paste goblins.

Reply:
I think this was not addressed yet.

Reply:
I'll double check, I may have missed one.

Reply:
Fixed